docs/docs/How-to-Contribute.md

```diff
@@ -0,0 +1,73 @@
+---
+id: How-to-Contribute
+title: How to Contribute
+---
+First of all, thank you for the interest in contributing to the Mentorship System!
+
+Any issue labeled as [Status: Available](https://github.com/anitab-org/mentorship-backend/issues?q=is%3Aissue+is%3Aopen+label%3A%22Status%3A+Available%22) or [First Timers Only](https://github.com/anitab-org/mentorship-backend/issues?q=is%3Aissue+is%3Aopen+label%3A%22First+Timers+Only%22) can be worked by any contributor.
+
+Issues labeled as `First Timers Only` are more oriented towards newcomers, with the exception of Quality Assurance issues that can be worked by any contributor because there can always be some bug and these issues are always available.
+
+There are many ways you can contribute to this project.
+- **Test the backend for Quality Assurance** purposes and **report bugs** (if any);
+- **Give feedback** on what is shared with the community (UI prototype, suggesting new features); 
+- **Participate in discussions** on Slack; 
+- **Solving the available issues**, implement features, fix bugs, develop tests, etc...;
+- **Helping others** understand the project **on Slack**;
+- Review pull requests (PRs), even if you're a newcomer.
+
+## Code
+
+You can help on the development. You can check the contribution guidelines for this, `CONTRIBUTING.md` file inside `.github` folder.
+
+Before starting to contribute make sure to check our [Commit Message Style Guide](https://github.com/anitab-org/mentorship-backend/wiki/Commit-Message-Style-Guide).
+
+You can check existing issues that are available and not assigned to anyone. You can filter also issues with labels such as [Category: Coding](https://github.com/anitab-org/mentorship-backend/issues?q=is%3Aissue+is%3Aopen+label%3A%22Category%3A+Coding%22) which are available [Status: Available](https://github.com/anitab-org/mentorship-backend/issues?q=is%3Aissue+is%3Aopen+label%3A%22Status%3A+Available%22).
+
+You can find a bugs while testing and submit an issue to solve them.
+
+You can find more about [Tech Stack here](https://github.com/anitab-org/mentorship-backend/wiki/Tech-Stack).
+
+## Quality Assurance
+
+You can test the backend for bugs. We have a set of [Quality Assurance Test Cases](https://github.com/anitab-org/mentorship-backend/wiki/Quality-Assurance) to help you with test the backend to lookup these bugs.
```

Comment:
here as well, I guess there are more changes related to the link part in the docs

Reply:
You could make this change wherever required

Reply:
You could make this change wherever required